src/components/InviteOptionsModal.tsx

```diff
@@ -1,56 +1,94 @@
+import { getPhoneHash } from '@celo/utils/lib/phoneNumbers'
```

Comment:
@jeanregisser i think this is safe to use? we need a way to track the success rate of invitations, and this was the only way i could think of without capturing any PII

Reply:
This look good, can't think of a better way 👍